app/src/main/res/layout/activity_conversations.xml

```diff
@@ -1,47 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  android:fitsSystemWindows="true"
-  tools:context=".Conversations">
-
-  <com.google.android.material.appbar.AppBarLayout
-    android:id="@+id/app_bar"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/app_bar_height"
-    android:fitsSystemWindows="true"
-    android:theme="@style/AppTheme.AppBarOverlay">
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    tools:context=".Conversations">
 
-    <com.google.android.material.appbar.CollapsingToolbarLayout
-      android:id="@+id/toolbar_layout"
+  <com.google.android.material.appbar.AppBarLayout
       android:layout_width="match_parent"
-      android:layout_height="match_parent"
-      android:fitsSystemWindows="true"
-      app:contentScrim="?attr/colorPrimary"
-      app:layout_scrollFlags="scroll|exitUntilCollapsed"
-      app:toolbarId="@+id/toolbar">
+      android:layout_height="wrap_content">
 
-      <androidx.appcompat.widget.Toolbar
+    <androidx.appcompat.widget.Toolbar
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        app:layout_collapseMode="pin"
-        app:popupTheme="@style/AppTheme.PopupOverlay" />
+        android:layout_height="wrap_content"
+        android:background="@color/colorPrimaryDark"
+        app:popupTheme="@style/MenuStyle">
 
-    </com.google.android.material.appbar.CollapsingToolbarLayout>
-  </com.google.android.material.appbar.AppBarLayout>
+      <de.hdodenhof.circleimageview.CircleImageView
+        android:layout_width="30dp"
+        android:layout_height="30dp"
+        android:id="@+id/profile_image"/>
 
-  <com.google.android.material.floatingactionbutton.FloatingActionButton
-    android:id="@+id/fab"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content"
-    android:layout_margin="@dimen/fab_margin"
-    app:layout_anchor="@id/app_bar"
-    app:layout_anchorGravity="bottom|end"
-    app:srcCompat="@android:drawable/ic_dialog_email" />
```

Comment:
Getting rid of the button will impact your new conversation page code, you might need to add another button elsewhere, maybe a similar floating action button in the bottom right corner @pfellows10014